index.bs

```diff
@@ -364,58 +369,62 @@ to ease the use of Suborigins in legacy applications are also presented. See
 ## Representation of Suborigins ## {#representation}
 
 At an abstract level, a suborigin consists of the <a>scheme</a>, <a>host</a>,
-and <a>port</a> of a traditional origin, plus a <a>suborigin namespace</a>.
+and <a>port</a> of a physical (traditional) origin, plus a <a>suborigin namespace</a>.
 However, as mentioned above, suborigins are intended to fit within the framework
 of [[!RFC6454]].  Therefore, this specification provides a way of serializing a
-Suborigin bound resource into a traditional Origin. This is done by inserting
+Suborigin bound resource into a physical (traditional) origin. This is done by inserting
 the suborigin namespace into the host of the Origin, thus creating a new host
 but maintaining all of the information about both the original scheme, host,
-port, and the suborigin namespace. This is done by prepending the host name with
+port, and the suborigin namespace. The serialization format prepends the host name with
 the suborigin namespace followed by a "`_`" character.
 
-For example, if the resource is hosted at `https://example.com/` in
-the suborigin namespace `profile`, this would be serialized as
+For example, a resource hosted at `https://example.com/` in
+the suborigin namespace `profile` would be serialized as
 `https://profile_example.com/`.
 
-Similarly, if a resource is hosted at `https://example.com:8080/` in
-the suborigin namespace `separate`, this would be serialized as
+Similarly, a resource hosted at `https://example.com:8080/` in
+the suborigin namespace `separate` would be serialized as
 `https://separate_example.com:8080/`.
 
-Internally, the <a>suborigin namespace</a> must be tracked by the user agent.
-When the origin needs to be serialized for a resource, the user agent should
+Internally, the user agent just track the <a>suborigin namespace</a> of the resource.
+When the origin needs to be serialized, the user agent should
 follow the algorithm in [[#serializing]].
 
+Note that the underscore character is not a valid character for hostnames used
+on the web [[!RFC1123]] and as a result, the serialization above cannot collide
```

Comment:
"cannot" is a bit strong; the Web is a scary and big place :-) Maybe we should rephrase as "should not" or, alternatively, "cannot collide with valid existing hostnames"?